renderdoc/driver/vulkan/vk_layer_android.cpp

```diff
@@ -78,10 +82,7 @@ VK_LAYER_EXPORT VkResult VKAPI_CALL vkEnumerateInstanceLayerProperties(uint32_t
 VK_LAYER_EXPORT VkResult VKAPI_CALL vkEnumerateInstanceExtensionProperties(
     const char *pLayerName, uint32_t *pPropertyCount, VkExtensionProperties *pProperties)
 {
-  // we don't export any instance extensions
-  if(pPropertyCount)
-    *pPropertyCount = 0;
-
-  return VK_SUCCESS;
+  return VK_LAYER_RENDERDOC_CaptureEnumerateInstanceExtensionProperties(
+      VK_NULL_HANDLE, pLayerName, pPropertyCount, pProperties);
```

Comment:
`VK_NULL_HANDLE` is for vulkan objects like `VkInstance` or `VkPhysicalDevice`, this should be just `NULL`.

Reply:
Ah I see, taken care of 👍